node/Cargo.toml

```diff
@@ -9,7 +9,7 @@ build = 'build.rs'
 edition = '2018'
 
 [[bin]]
-name = 'moonbeam'
+name = 'sunrock'
```

Comment:
Don't change the binary name, not neede and it will break many parts of the CI / deployment